llpc/context/llpcCompiler.cpp

```diff
@@ -2126,6 +2157,7 @@ void Compiler::LinkRelocatableShaderElf(
     {
         ElfReader<Elf64> vsReader(m_gfxIp);
         ElfReader<Elf64> fsReader(m_gfxIp);
+        ElfReader<Elf64> fchReader(m_gfxIp);
         if (!pShaderElfs[ShaderStageVertex].empty())
```

Comment:
I didn't realize this before, but vertex shaders are non-optional in graphics pipelines, so... maybe change this condition to an assertion in a separate change?